### PR TITLE
Added minutes & action list

### PR DESCRIPTION
**Pull request by annefou**
Please review the minutes of our last meeting & Action list.

Anne.

### CODE REVIEW

File: meetings/160330_organisers.md [hunk 1]
```diff
@@ -1,11 +1,172 @@
...
+### Evaluation ###
+
+We all had very positive comments on the Carpentry week. More than 100 persons attended the workshops and we recruted new helpers and instructors.
+
+A special thank to the Library who was fantastic (food, registration, coffee, stickers, etc.).
+
+A meeting is planned on Wednesday 31th of March with the Library & Arnstein for an evaluation of the Carpentry week and planning for next events. 
+
+Lex asked if an instructor from the Software Carpentry workshop and one from the Spatio-temporal Data arpentry workshop could attend it.
+
```
Comment (karinlag) [inline meetings/160330_organisers.md]: Anne and Halvdan agree to come along.

File: meetings/160330_organisers.md [hunk 2]
```diff
@@ -1,11 +1,172 @@
...
+Anne said we poorly advertised the spatio-temporal Data Carpentry workshop: we initially advertised it as a Data Carpentry workshop for Geoscientists while using spatio-temporal data is not specific to geoscientists.
+
+Lex also believes the reproducibility workshop was not well advertised too.
+
+As an overall comment, we agreed to be more careful when we advertise Data Carpentry workshops and focus more on the lesson contents rather then on the Department organizer.
+
+Several python issues with matplotlib were encountered during Software Carpentry workshop: several attendees could not import matplotlib even though they all installed python anaconda. Arvind has similar problems too.
+
+
+Hugues mentionned the difficulties they had to get positive and negative inputs written on stickers: they asked for feedbacks at the end of each session but got very little feedbacks. Karin said that if it does not work with stickers, it can be done orally (Greg Wilson does it very often and it works very well according someone takes notes of the comments!).
```
Comment (karinlag) [inline meetings/160330_organisers.md]: Lex mentioned the thing with stickers that Greg did, not me.

File: meetings/160330_organisers.md [hunk 3]
```diff
@@ -1,11 +1,172 @@
...
+
+##Adoption of Agenda##
+
+The agenda was adopted and a new item on github has been added at Michel's request.
+
+## Carpentry Week: Evaluation and plans ##
+
+### Evaluation ###
+
+We all had very positive comments on the Carpentry week. More than 100 persons attended the workshops and we recruted new helpers and instructors.
```
Comment (lexnederbragt) [inline meetings/160330_organisers.md]: recruited

File: meetings/160330_organisers.md [hunk 4]
```diff
@@ -1,11 +1,172 @@
...
+- 1/2 day on Shell where we could teach the entire Software Carpentry Lesson
+- 1/2 day on licensing (software and data license)
+- 1/2 or 1 day introducing Python or R (could be useful for Data Carpentry follow-up workshops)
+- 1/2 or 1 day Software Carpentry workshop for professors
+
+Vincent also mentionned the interest of ForBio (http://www.forbio.uio.no) to develop a Software Carpentry workshop week. The ambition of Forbio is to train a new generation of biosystematists in the Nordic countries and as many PhDs are coming from abroad, it would be necessary to hold a full week rather than several 1/2 days or 1 day workshops. 
+
+We discussed the possibility to participate more actively in Research school to train younger researchers (PhDs), master Students or even undergraduates. We all agreed there is a need to adapt Software Carpentry workshop "format".
+
+The codata international Training Workshop format was discussed: they offer the possibility to build your own workshop with lessons chosen according to your needs. We all think we could use the same kind of format, with adequate information on the possible prerequisites for each session.
```
Comment (lexnederbragt) [inline meetings/160330_organisers.md]: You could link to and example workshop: http://www.codata.org/news/105/62/CODATA-International-Training-Workshop-in-Big-Data-for-Science-Beijing-4-17-July

File: meetings/160330_organisers.md [hunk 5]
```diff
@@ -1,11 +1,172 @@
...
+Karin will set-up this subgroup and organize a meeting to discuss it. Vincent, Carrie and Anne already expressed their interests.
+
+## git vs github ##
+
+
+Michel asks for more training on github (not only on git) and also mentionned the necessity to train the trainers. 
+
+On the github part, Lex said we could even develop a more github oriented lesson for git where the usage of command line would not be necessary.
+
+About the necessity to train the trainers, we have started to write some documents on how to organize a workshop at UIO or on who can teach what and we think we should develop more UIO-based information.
```
Comment (lexnederbragt) [inline meetings/160330_organisers.md]: Not sure I follow you here. We are looking into organising an instructor training workshop at UiO. I am not sure about what you mean with the other points...

File: meetings/160330_organisers.md (hunk 2 above)
Comment (lexnederbragt) [inline meetings/160330_organisers.md]: mentioned (typo, also elsewhere)

File: meetings/160330_organisers.md (hunk 5 above)
Comment (annefou) [inline meetings/160330_organisers.md]: Ok, I was not sure I understood it properly...  What about changing the whole section by:

## Instructor training ##


Michel asks for more training on github (not only on git) and also mentioned his wish to get train and become an instructor.

For github, Lex suggested to look at   the Mozilla Science Lab website (https://www.mozillascience.org).

About the necessity to train the trainers, we are looking into organizing an instructor training workshop at UIO. Both Karin and Lex are in the process of becoming certified instructors for the instructor training program.

#### Actions ####

Lex will send some web links for github training.

Comment (lexnederbragt) [inline meetings/160330_organisers.md]: Yes, good! The mozilla link is https://www.mozillascience.org/making-friends-with-github.

**Pull request merged**